sonaranalyzer-dotnet/src/Tests/SonarAnalyzer.UnitTest/TestCases/DoNotCheckZeroSizeCollection.cs

```diff
@@ -92,7 +92,7 @@ public void TestCountProperty()
 //                        ^^^^^^^^^^^^^^^^^^^^^^^^^
 
             var nonCollection = new FooProperty();
-            result = nonCollection.Count() >= 0;
+            result = nonCollection.Count >= 0;
```

Comment:
Why this change?

Reply:
It was invalid code, there was no method, but the property (as FooProperty suggests). I only found it when translating the code to VB.